shared/src/main/scala/org/atnos/eff/FutureEffect.scala

```diff
@@ -24,6 +24,15 @@ trait FutureCreation {
 
   def async[R, A](a: =>A)(implicit m: Member[Future, R], ec: ExecutionContext): Eff[R, A] =
     send(Future(a))
+
+  def lift[R, A](
```

Comment:
Also in terms of coding style I generally put everything on the same line unless it goes above 120 chars. And I don't use accolades for 1-line statements.